internal/criocli/criocli.go

```diff
@@ -777,6 +780,13 @@ func getCrioFlags(defConf *libconfig.Config) []cli.Flag {
 			EnvVars:   []string{"CONTAINER_VERSION_FILE_PERSIST"},
 			TakesFile: true,
 		},
+		&cli.StringFlag{
+			Name:      "clean-shutdown-file",
+			Usage:     "Location for CRI-O to lay down the clean shutdown file. It indicates whether we've had time to sync changes to disk before shutting down. If not found, crio wipe will clear the storage directory",
```

Comment:
The verbiage is awesome, but is it too long for the usage?  I'd a vague memory that we wanted (required?) these to be under 80 chars.